lte/protos/session_manager.proto

```diff
@@ -792,33 +816,54 @@ enum SMSessionFSMState {
 //PDU session Establishment from AMF to SMF
 //SET MESSSAGE
 message M5GSMSessionContext {
-  bytes pdu_session_id = 1;   //One byte
-  RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
-  RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+  uint32 pdu_session_id = 1;
+ //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
```

Comment:
All comments should have a space after the `//`
```suggestion
 // Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
```

Reply:
Changes done for both proto files.